src/utils/uncompletable-helpers.test.ts

```diff
@@ -7,124 +7,42 @@ import {
-        })
-
-        test('handles empty string', () => {
-            expect(addUncompletablePrefix('')).toBe('* ')
-        })
-
-        test('handles content with just asterisk (no space)', () => {
-            expect(addUncompletablePrefix('*No space')).toBe('* *No space')
-        })
-
-        test('handles content with multiple asterisks', () => {
-            expect(addUncompletablePrefix('** Bold text')).toBe('* ** Bold text')
+        test.each([
+            ['Task content', '* Task content'],
+            ['* Already prefixed', '* Already prefixed'],
+            ['', '* '],
+            ['*No space', '* *No space'],
+            ['** Bold text', '* ** Bold text'],
+        ])('transforms "%s" to "%s"', (input, expected) => {
+            expect(addUncompletablePrefix(input)).toBe(expected)
         })
     })
 
     describe('removeUncompletablePrefix', () => {
-        test('removes prefix from content with prefix', () => {
-            expect(removeUncompletablePrefix('* Task content')).toBe('Task content')
-        })
-
-        test('does not modify content without prefix', () => {
-            expect(removeUncompletablePrefix('Regular task')).toBe('Regular task')
-        })
-
-        test('handles content with just prefix', () => {
-            expect(removeUncompletablePrefix('* ')).toBe('')
-        })
-
-        test('does not remove asterisk without space', () => {
-            expect(removeUncompletablePrefix('*No space')).toBe('*No space')
-        })
-
-        test('handles content with multiple prefixes', () => {
-            expect(removeUncompletablePrefix('* * Double prefix')).toBe('* Double prefix')
+        test.each([
+            ['* Task content', 'Task content'],
+            ['Regular task', 'Regular task'],
+            ['* ', ''],
+            ['*No space', '*No space'],
+            ['* * Double prefix', '* Double prefix'],
+        ])('transforms "%s" to "%s"', (input, expected) => {
+            expect(removeUncompletablePrefix(input)).toBe(expected)
         })
     })
 
     describe('hasUncompletablePrefix', () => {
-        test('returns true for content with prefix', () => {
-            expect(hasUncompletablePrefix('* Task content')).toBe(true)
-        })
-
-        test('returns false for content without prefix', () => {
-            expect(hasUncompletablePrefix('Regular task')).toBe(false)
-        })
-
-        test('returns false for asterisk without space', () => {
-            expect(hasUncompletablePrefix('*No space')).toBe(false)
-        })
-
-        test('returns true for just the prefix', () => {
-            expect(hasUncompletablePrefix('* ')).toBe(true)
-        })
-
-        test('returns false for empty string', () => {
-            expect(hasUncompletablePrefix('')).toBe(false)
+        test.each([
+            ['* Task content', true],
+            ['Regular task', false],
+            ['*No space', false],
+            ['* ', true],
```

Comment:
Tested this case in the app and a task with content '* ' is not considered uncompletable.